read/parser.go

```diff
@@ -126,9 +126,8 @@ type Dump struct {
 	// map from type address to type
 	TypeMap map[uint64]*Type
 
-	// map from itab address whether the data field of an iface
-	// with that itab contains a pointer.
-	ItabMap map[uint64]bool
+	// map from the address of an interface to the address of the type
```

Comment:
Substitute "an interface" with "an itab".
It's not the address of the interface itself.